src/NuGetPackageVerifier/third-party.json

```diff
@@ -0,0 +1,74 @@
+﻿{
+  "prefixWhitelist": [
+    "Libuv"
+    "Microsoft.AspNetCore",
+    "Microsoft.CodeAnalysis.Razor",
+    "Microsoft.Data.Sqlite",
+    "Microsoft.EntityFrameworkCore",
+    "Microsoft.Extensions.",
+    "Microsoft.Net.Http.Headers",
+    "Microsoft.NETCore.App",
+    "Microsoft.Owin.Security.Interop",
+    "Microsoft.VisualStudio.Web.",
+    "NETStandard.Library",
+    "System.",
+  ],
+  "prefixBlacklist":  [
+    "System.Ben"
+  ],
+
+  "packages": {
+    "Microsoft.ApplicationInsights.AspNetCore": [ "2.0.0" ],
+    "Microsoft.AspNet.Identity.EntityFramework": [ "2.2.1" ],
+    "Microsoft.AspNet.WebApi.Client": [ "5.2.2" ],
+    "Microsoft.Azure.KeyVault": [ "2.0.6" ],
+    "Microsoft.Build.Runtime": [ "15.2.0-preview-000047-02" ],
```

Comment:
Can we use floating versions? `15.2.0-*`? This is likely to change frequently, even daily.